src/app/pages/home/components/home-tabs.tsx

```diff
@@ -1,66 +1,51 @@
-import { Suspense } from 'react';
+import { Suspense, useCallback, useMemo } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
 
-import { Box, Flex, SlideFade, Stack } from '@stacks/ui';
+import { Box, Stack } from '@stacks/ui';
 import type { StackProps } from '@stacks/ui';
 
-import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
-import { useHomeTabs } from '@app/common/hooks/use-home-tabs';
+import { RouteUrls } from '@shared/route-urls';
+
 import { LoadingSpinner } from '@app/components/loading-spinner';
 import { Tabs } from '@app/components/tabs';
 
 interface HomeTabsProps extends StackProps {
-  balances: React.JSX.Element;
-  activity: React.JSX.Element;
+  children: React.JSX.Element;
 }
+// TODO: Abstract this to generic RouteTab once choose-fee-tab updated
+export function HomeTabs({ children }: HomeTabsProps) {
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
+
+  const tabs = useMemo(
+    () => [
+      { slug: RouteUrls.Home, label: 'balances' },
+      { slug: RouteUrls.Activities, label: 'activity' },
```

Comment:
Just curious here, why plural? `RouteUrls.Activity`?

Reply:
It's an oversight, I'll change it to `RouteUrls.Activity`, thanks! 👍